artemis/reporting/modules/ssh_bruter/reporter.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Any, Dict, List
+
+from artemis.modules.ssh_bruter import SSHBruterResult
+from artemis.reporting.base.language import Language
+from artemis.reporting.base.report import Report
+from artemis.reporting.base.report_type import ReportType
+from artemis.reporting.base.reporter import Reporter
+from artemis.reporting.base.templating import ReportEmailTemplateFragment
+from artemis.reporting.utils import get_top_level_target
+from artemis.resolvers import IPResolutionException, ip_lookup
+
+
+class SSHBruterReporter(Reporter):
+    EXPOSED_SSH_WITH_EASY_PASSWORD = ReportType("exposed_ssh_with_easy_password")
+
+    @staticmethod
+    def create_reports(task_result: Dict[str, Any], language: Language) -> List[Report]:
+        if task_result["headers"]["receiver"] != "ssh_bruter":
+            return []
+
+        if not task_result["status"] == "INTERESTING":
+            return []
+
+        if not isinstance(task_result["result"], SSHBruterResult):
```

Comment:
when deserializing from db, it wouldn't be a SSHBruterResult instance, but a dict